analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -770,6 +768,8 @@ def signal_handler(signum, frame):
         LOG.debug_analyzer("%s is skipped", skp.source)
 
     LOG.info("Total analyzed compilation commands: %d", len(analyzed_actions))
+    if (any(skipped_actions)):
```

Comment:
Wouldn't this be enough?
```suggestion
    if skipped_actions:
```